test/fixtures/assignments.yml

```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Would be good to replace standard fixtures by factories from gem [factory_bot_rails](https://github.com/thoughtbot/factory_bot_rails#factory_bot_rails---)

Could be done later.